README.md

```diff
@@ -1,2 +1,88 @@
-# Library
-A mini book library app
+# LIBRARY
+
+<!--
+*** Thanks for checking out this README Template. If you have a suggestion that would
+*** make this better, please fork the repo and create a pull request or simply open
+*** an issue with the tag "enhancement".
+*** Thanks again! Now go create something AMAZING! :D
+-->
+
+<!-- PROJECT SHIELDS -->
+
+<p align="center">
+  <a href="https://github.com/ebeagusamuel/Library">
+    <img src="img/microverse.png" alt="Microverse Logo" width="80" height="80">
+  </a>
+
+  <p align="center">
+    This project is part of the Microverse curriculum in JavaScript module!
+  </p>
+</p>
+
+<!-- TABLE OF CONTENTS -->
+
+## Table of Contents
+
+- [About the Project](#about-the-project)
+- [Contributors](#contributors)
+- [Acknowledgements](#acknowledgements)
+- [License](#license)
+
```

Comment:
Just one more thing.
let's make this repo look professional by adding live link and screenshot of your app.
It would be nice to add a live link, according to the content on the pathwright we should have a live link.
Kindly checkout Microverse [template](https://github.com/microverseinc/readme-template) from pathwright
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/